frontend/src/pages/HomePage/index.js

```diff
@@ -18,11 +18,11 @@ const HomePage = () => {
 
       <div className={styles.container}>
         <div className={styles.informationBlock}>
-          <Text>
+          <Body>
             Spin the {WHEEL_CONFIG.name} to win merch, experiences, NFTs and
             more.
-          </Text>
-          <Text>What are you waiting for? Let&apos;s get started!</Text>
+          </Body>
+          <Body>What are you waiting for? Let&apos;s get started!</Body>
```

Comment:
Can you explain why you chose to go with `body` 

I'm asking because there's an HTML element called `body`. Usually you'll only find one of those, so seeing multiple bodies is a bit confusing for me. But also, as a general name choice for a component, it doesn't tell me much. In fact, even if you went with `Body` I would've probably expected to see layout styles applied.

Reply:
I think `Text` was quite okay?

Reply:
@mirhamasala This was to match the naming Hanna has [over here](https://www.figma.com/file/36e7nnrNlYQENyigL9sQED/Wheel-of-Fortune?node-id=496%3A2)

Reply:
Yes that was what I aimed for. That said we all agree that I should rename them `Text` again?

Reply:
@joaojusto Want to keep it then?

Reply:
Besides the naming question. Do we need to use two `<Body>` components here, though?